docs/sources/alerting/notifications.md

```diff
@@ -230,3 +230,9 @@ Notification services which need public image access are marked as 'external onl
 
 All alert notifications contain a link back to the triggered alert in the Grafana instance.
 This URL is based on the [domain]({{< relref "../administration/configuration/#domain" >}}) setting in Grafana.
+
+## Notification templating
+
+> **Note:** Alert notification templating is only available in Grafana v7.4 and above.
+
+You can [inject alert query labels into an alert notification]({{< relref "./add-notification-template.md" >}}) by referring to the matching alert condition query's [labels]({{< relref "../getting-started/timeseries-dimensions.md#labels" >}}) using the `${Label}` syntax.
```

Comment:
It is a long and confusing sentence. Can we simplify into 2 shorter and clear sentences.